src/main/java/api/weather/DayTemperatureInfo.java

```diff
@@ -0,0 +1,11 @@
+package api.weather;
+
+import java.time.Month;
+
+public interface DayTemperatureInfo {

```

Comment:
Please, remove the 3rd task from this review. 
See https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History
You need to remove the last commit from your `master` branch